utilities/file-to-string/src/file-to-string/file-to-string.ts

```diff
@@ -0,0 +1,48 @@
+import { existsSync, mkdirSync, readdirSync, rmSync } from 'fs';
+import { readFile, writeFile } from 'fs/promises';
+import { join } from 'path';
+
+import type { Config, FileToStringArgs } from './file-to-string.types.js';
+
+const DEFAULT_OUTPUT_DIR = '__raw__';
+const DEFAULT_EXTENSION = 'ts';
+
+const cleanDirectory = (dir: string) => {
+  if (!existsSync(dir)) {
+    return;
+  }
+
+  readdirSync(dir).forEach((file) => rmSync(join(dir, file)));
```

Comment:
Can you just recursively clean instead?

Reply:
`{ recursive: true}` in the options should do it (assuming that's what you want)

Reply:
Added 👍